common/utils/src/main/resources/error/error-classes.json

```diff
@@ -2060,6 +2085,74 @@
     },
     "sqlState" : "42000"
   },
+  "INVALID_INTERVAL_FORMAT" : {
+    "message" : [
+      "Error parsing '<input>' to interval. Please ensure that the value provided is in a valid format for defining an interval. You can reference the documentation for the correct format."
+    ],
+    "subClass" : {
+      "ARITHMETIC_EXCEPTION" : {
+        "message" : [
+          "Uncaught arithmetic exception: <e>."
```

Comment:
Since this is an arithmetic error in interval argument, arbitrary `<e>` is removed. Message now provides information that an uncaught arithmetic error happened.